lib/Descriptors.h

```diff
@@ -42,14 +42,14 @@
 #include "lufa/LUFA/Drivers/USB/USB.h"
 
 /* Macros: */
-/** Endpoint address of the CDC device-to-host notification IN endpoint. */
-#define CDC_NOTIFICATION_EPADDR (ENDPOINT_DIR_IN | 2)
-
 /** Endpoint address of the CDC device-to-host data IN endpoint. */
-#define CDC_TX_EPADDR (ENDPOINT_DIR_IN | 3)
+#define CDC_TX_EPADDR (ENDPOINT_DIR_IN | 1)
```

Comment:
:thinking: just a thought - can't we run into mysterious issues with flashing the MMU board on some platforms? Hopefully not, USB arbitration should be hidden from the avr-dude, but who knows?

Reply:
Yes, the endpoint structure is hidden from avrdude. It's the OS CDC class driver that handles this. As for whether it can be a problem, I very much doubt it. There is no hardcoded order in which endpoints have to be allocated, so any driver should handle this. The only maybe risky driver is the custom one on Windows that applies the MMU product name instead of a generic name to the virtual com port, but that seems to be working just fine on my end, so I doubt there will be any issue caused by this.